compiler/angkor/README.md

```diff
@@ -2,11 +2,11 @@
 
 ## Purpose
 
-_angkor_ is a `nncc` core library
+_angkor_ is an `nncc` core library
```

Comment:
why `an` ?

Reply:
found that `nncc` is pronounced like `en en see see` so it would be better with `an` , not `a`

Reply:
Yes, that's exactly the rule for "a" vs "an" in English. It only matters how you pronounce the following word, not how you spell it.
For example: an XML ("eks em el") vs a XAML ("a zammel").